rust/src/web/graphql/transactions/mod.rs

```diff
@@ -96,16 +98,12 @@ impl TransactionsQueryRoot {
 fn txn_from_hash(cmd: SignedCommandWithData, db: &Arc<IndexerStore>) -> Transaction {
     let block_state_hash = cmd.state_hash.to_owned();
     let block_date_time = date_time_to_scalar(cmd.date_time as i64);
-
-    let canonical = match db
-        .get_block_canonicity(&block_state_hash.to_owned())
-        .unwrap()
-    {
-        Some(canonicity) => canonicity == Canonicity::Canonical,
-        None => false,
-    };
-
-    Transaction::from_cmd(cmd, block_date_time, &block_state_hash, canonical)
+    Transaction::from_cmd(
+        cmd,
+        block_date_time,
+        &block_state_hash,
+        get_block_canonicity(db, &block_state_hash.0),
+    )
```

Comment:
Just inlining the call to `get_block_canonicity`; makes sense.

Reply:
There were several versions of `get_block_canonicity` in the GQL modules. Those all got refactored to this common version